NetKAN/QuodiosKermansMineToEarn.netkan

```diff
@@ -0,0 +1,7 @@
+{
+    "$kref": "#/ckan/spacedock/951",
+    "identifier": "QuodiosKermansMineToEarn",
+    "spec_version": "v1.4",
+    "license": "GPLv3",
```

Comment:
The license string is not valid according to the [Debian specification](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification) that [NetKAN uses](https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md#license). It needs to be changed from "GPLv3" to "GPL-3.0". I believe that this can either be changed in the license field on SpaceDock (not recommended since GPLv3 is the most common abbreviation, and would only work if SpaceDock submits a new pull request when metadata is changed, I'm uncertain), or an additional NetKAN file could be created that overrides or replaces the auto-generated field.